README.md

```diff
@@ -16,14 +16,34 @@ Usage
 1. [Install Ansible](http://docs.ansible.com/ansible/intro_installation.html) on your local computer and verify that it works
     - ansible --version
     - note: on MacOS Ansible might not be added to your PATH in bash_profile, see [2nd answer in this thread](https://stackoverflow.com/questions/35898734/pip-installs-packages-successfully-but-executables-not-found-from-command-line/35899029) on how to fixed that.
-1. [Initialize a memory card](https://www.raspberrypi.org/documentation/installation/installing-images/) (on your local computer) with the latest [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/) with [Etcher](https://www.balena.io/etcher/) or from the command line:
-    - `diskutil list`
-    - `diskutil unmountDisk /dev/<disk#>`
-    - `sudo dd bs=1m if=<your image file>.img of=/dev/<disk#>`
-1. [Enable ssh](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/) for Raspbian headless operation (while memory card is still attached to your local computer)
-    - `touch /Volumes/boot/ssh`
-1. Insert the memory card in the to-be-provisioned Raspberry Pi and connect the it to the local network.
-1. Clone this git repository on your local computer (if you haven't already done so)
+1. Install Raspberry OS Lite on an SD-card with your local computer.
+    - Download the [Raspberry Pi Imager] (https://www.raspberrypi.org/software/) and install the OS (the tool will download the latest image)
+    - Alternatively you can download the image yourself and write this to the SD-card with a tool like Belana Etcher or directly from the command line with the following commands:
+        ```
+        diskutil list
+        diskutil unmountDisk /dev/<disk#>
+        sudo dd bs=1m if=<your image file>.img of=/dev/<disk#>
+        ```
+1. [Enable SSH](https://www.raspberrypi.org/documentation/remote-access/ssh/) while the SD-Card is still in the local computer.
+    - Windows: create an empty file `ssh` on the SD-card (which now has the volume label 'boot)
+    - Linux/MacOS: `touch /Volumes/boot/ssh`
+1. (Optionally) Configure Raspberry Pi to [connect to your Wifi network](https://www.raspberrypi.org/documentation/configuration/wireless/headless.md) (otherwise you have to connect through Ethernet)
+    - Create a file `wpa_supplicants.conf` on the SD-card with the following content:
+        ```
+        ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+        update_config=1
+        country=<Insert 2 letter ISO 3166-1 country code here>
+
+        network={
+        ssid="<Name of your wireless LAN>"
+        psk="<Password for your wireless LAN>"
+        }
+       ```
+1. Test Raspberry OS and connection.
+    - Insert the memory card in the to-be-provisioned Raspberry Pi and connect the it to the local network.
+    - ssh into your Pi with [standard credentials](https://www.raspberrypi.org/documentation/linux/usage/users.md) (username=pi, password=raspberry):
+            `ssh <ip> -lpi`
```

Comment:
`-l pi` isn't there a missing space there?

Reply:
Tested, both are possible. Will add the space for clarity.